### PR TITLE
Initial (very simple) contribution of PL/SQL highlighting rules

### PR DESCRIPTION
**Pull request by chezwicker**
Adding support for PL/SQL (Oracle)

### CODE REVIEW

File: lib/ace/ext/modelist.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ var supportedModes = {
     Pascal:      ["pas|p"],
     Perl:        ["pl|pm"],
     pgSQL:       ["pgsql"],
+    PL/SQL:      ["PL/SQL"],
```
Comment (nightwing) [inline lib/ace/ext/modelist.js]: this should have name of the mode and common extensions used for that mode. something like `plSQL: ["pls"] `

File: lib/ace/mode/plsql.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * ***** END LICENSE BLOCK ***** */
+
+define(
+	function(require, exports, module) {
```
Comment (nightwing) [inline lib/ace/mode/plsql.js]: Please use formatting similar to other modes: indent with 4 spaces do not add newline or tab after (

**Comment by prespic** [comment]
Hi, any chance to finish this merge request? PL/SQL would be fine.

**Comment by akoreman** [comment]
A highlight mode for PL/SQL has been added in a different PR (#5037) since this PR was opened, I'll go ahead and close this PR.

**Pull request closed without merging**